server-common/src/main/resources/jcasbin_model.conf

```diff
@@ -0,0 +1,56 @@
+; Licensed to the Apache Software Foundation (ASF) under one
+; or more contributor license agreements.  See the NOTICE file
+; distributed with this work for additional information
+; regarding copyright ownership.  The ASF licenses this file
+; to you under the Apache License, Version 2.0 (the
+; "License"); you may not use this file except in compliance
+; with the License.  You may obtain a copy of the License at
+;
+;  http://www.apache.org/licenses/LICENSE-2.0
+;
+; Unless required by applicable law or agreed to in writing,
+; software distributed under the License is distributed on an
+; "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+; KIND, either express or implied.  See the License for the
+; specific language governing permissions and limitations
+; under the License.
+
+; The model file of jcasbin defines a permission model used to describe the permissions
+; of users, groups, and roles for a single metadata type.
+
+; "r" represents the parameters passed when making a request to jCasbin.
```

Comment:
Do we need to consider the group authorization here?

Reply:
jcasbin does not distinguish between roles, groups, and users; they are all considered as "subject" in its context. In practice, we can simply pass only the user when using it.

Reply:
OK, we should add a test case at least.

Reply:
It's already there.

<img width="615" alt="image" src="https://github.com/user-attachments/assets/27ca0136-e638-41b6-b5f5-5ed04b0b63d0" />

Reply:
OK, I got it.